charts/spire/charts/spiffe-csi-driver/README.md

```diff
@@ -52,8 +52,7 @@ A Helm chart to install the SPIFFE CSI driver.
 | `serviceAccount.name`                         | The name of the service account to use. If not set and create is true, a name is generated.                                                                              | `""`                                        |
 | `podAnnotations`                              | Pod annotations for spiffe-csi-driver                                                                                                                                    | `{}`                                        |
 | `podSecurityContext`                          | Security context for CSI driver pods                                                                                                                                     | `{}`                                        |
-| `securityContext.readOnlyRootFilesystem`      | Flag for read only root filesystem                                                                                                                                       | `true`                                      |
-| `securityContext.privileged`                  | Flag for specifying privileged mode                                                                                                                                      | `true`                                      |
+| `securityContext`                             | Security context                                                                                                                                    | `true`                                      |
```

Comment:
I dont understand this change here. Why not keep these values but just make sure they are used?

Reply:
Thanks for explaining i get it now. Please continue

Reply:
Done.

Reply:
Thank you.